src/sections/Meshery/Meshery-platforms/index.js

```diff
@@ -42,6 +42,10 @@ const supported_platforms = [
         <Code codeString={dedent`curl -L https://meshery.io/install | PLATFORM=docker bash -`
         }
         />
+        <h2 style={{ marginTop: "20px" }}>Docker Extension</h2>
+        <Code codeString={dedent`docker extension install meshery/docker-extension-meshery:stable-latest`
```

Comment:
@ranjanmangla1 We've a dedicated page for Meshery docker extension: https://layer5.io/docker-extension-meshery, can you include a link to it here. 
for example: "know more about Meshery docker extension" and then the link

Reply:
So, I should remove the installation command instead and add the link instead?

Reply:
@ranjanmangla1 NO, add the text redirecting to the page, between the heading and command.

Reply:
Great suggestion to tie in a link to the dedicated Extensions page https://layer5.io/docker-extension-meshery. Let's forego its inclusion here and seek out other pages to interlink. 👍